Requirements/Application-Requirements.md

```diff
@@ -0,0 +1,28 @@
+# Application Requirements
+
+The following requirements were decided upon to define what OpenTelemetry (OTel)
+signals the application will produce & when support for future SDKs should be
+added:
+
+1. Every supported language that has a GA Traces or Metrics SDK must have at
+   least 1 service example.
+
+    * Mobile support (Swift) is not an initial priority and not included in the
+      above requirement.
+
+2. Application processes must be language independent.
+
+    * gRPC is preferred where available and HTTP is to be used where it is not.
+
+3. Services should be architected to be modular components that can be switched out.
+
+    * Individual services can and should be encouraged to have multiple language
+      options available.
```

Comment:
This is more of a stretch goal right? First and foremost we should focus on achieving step 1 in this document.

Reply:
Yes more of a stretch goal. I think we'll inherently be a bit limited around the gRPC & HTTP usage but a more mature future version should have more language options per service within reason. But it's not a hard requirement to have at least 2 per service.